src/api/route-services/access/index.js

```diff
@@ -99,6 +99,74 @@ class AccessService {
 
     return projectIds;
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  async getExperimentEntries(table, experimentId) {
+    const docClient = new AWS.DynamoDB.DocumentClient({
+      region: config.awsRegion,
+    });
+    const getParams = {
+      TableName: table,
+      KeyConditionExpression: 'experimentId = :hkey',
+      ExpressionAttributeValues: {
+        ':hkey': experimentId,
+      },
+    };
+
+    const response = await docClient.query(getParams).promise();
```

Comment:
do we need to worry about pagination here?

Reply:
no, query returns up to 1Mb of data per partition key. We won't have more than a few rows per experimentID and anyway this will go away with SQL